portals/new_admin/portals/routes/users.js

```diff
@@ -0,0 +1,9 @@
+var express = require('express');
+var router = express.Router();
+
+/* GET users listing. */
+router.get('/', function(req, res, next) {
+  res.send('respond with a resourceeezzz333');
```

Comment:
Why have we used `resourceeezzz333` here? Was this done as a demo?

Reply:
Yes, here this was done as a demo.